docs/source/models/causes/opioid_use_disorder/index.rst

```diff
@@ -0,0 +1,822 @@
+
+Simulation Modeling Approaches for OUD
+---------------------------------------
+
+The opioid overdose crisis is driven by an intersecting set of social, structural, and economic forces. Simulation models are a valuable tool to help us understand and address this complex, dynamic, and nonlinear social phenomenon. [Cerda-2022]_ A systematic review of simulation models for opioid use and overdose identified several modeling frameworks used in this domain:
+
+- **Compartmental models** (36% of opioid epidemic models): Track population flows between defined health states (e.g., susceptible, with OUD, on treatment). Most commonly used approach for modeling OUD dynamics at the population level.
+
+- **Markov models** (20% of models): Account for state transitions with memory, allowing transition probabilities to depend on prior states or time spent in current state.
+
+- **System dynamics models** (16% of models): Capture feedback loops and dynamic interactions between substance use patterns, treatment capacity, and policy responses.
+
+- **Agent-based models** (16% of models): Represent individual-level heterogeneity in trajectories, allowing for complex interactions and emergent population-level patterns.
+
+The modeling framework described in this documentation uses a **compartmental model** approach (three-state model: Susceptible, With Condition, On Treatment), which is well-suited for integrating with GBD 2023 estimates and simulating population-level intervention effects. Key methodological considerations for all opioid simulation models include investment in model calibration and validation, transparency in assumptions and mechanics to facilitate reproducibility, and careful attention to potential bias in the choice of parameter inputs.
+
+GBD 2023 Modeling Strategy
++++++++++++++++++++++++++++
+
+GBD 2023 Definition and Diagnostic Criteria
+--------------------------------------------
+
+The Global Burden of Disease (GBD) 2023 study defines opioid use disorders as "a maladaptive pattern of opioid abuse, leading to clinically significant impairment or distress that includes symptoms of dependence, such as withdrawal symptoms or progressive tolerance." [GBD-2023-Overview]_ The GBD disease modeling includes cases that meet the diagnostic criteria for opioid dependence as defined by the Diagnostic and Statistical Manual of Mental Disorders, 4th edition, text revision (DSM-IV-TR) and the International Classification of Diseases (ICD-9 and ICD-10).
+
+**Diagnostic Codes**: The GBD disease modeling includes cases that meet the diagnostic criteria for opioid dependence as defined by DSM-IV-TR and ICD-9/ICD-10 substance use codes, primarily ICD-9 304.0 (opioid dependence) and 305.5 (opioid abuse) and ICD-10 F11.x (mental and behavioral disorders due to use of opioids). Opioid poisoning codes (e.g., ICD-9 965.0, E850.0-E850.2; ICD-10 T40.x) are used in overdose models rather than the chronic OUD cause.
```

Comment:
```suggestion
```
The diagnostic codes in the supplementary appendix have some surprises, which AI has ironed out at this point.  I don't think it is necessary to highlight them here.